docs/getting-started-guide.md

```diff
@@ -1,12 +1,303 @@
+The authenticity of host '192.168.99.100 (192.168.99.100)' can't be established.
+ECDSA key fingerprint is SHA256:algjLoZxaREX+nNzCQc0UWkNGuAQ7o6kxwMJBELZffg.
+Are you sure you want to continue connecting (yes/no)? yes
+Warning: Permanently added '192.168.99.100' (ECDSA) to the list of known hosts.
+rkt                                                                                                                                        100%   20MB  62.2MB/s   00:00
+stage1-coreos.aci                                                                                                                             100%   56MB  89.3MB/s   00:00
+rktlet                                                                                                                                     100%   29MB  66.3MB/s   00:00
+```
+
+Then, you need to ssh into the minikube VM and do some changes.
+
+```shell
+$ minikube ssh
+                         _             _
+            _         _ ( )           ( )
+  ___ ___  (_)  ___  (_)| |/')  _   _ | |_      __
+/' _ ` _ `\| |/' _ `\| || , <  ( ) ( )| '_`\  /'__`\
+| ( ) ( ) || || ( ) || || |\`\ | (_) || |_) )(  ___/
+(_) (_) (_)(_)(_) (_)(_)(_) (_)`\___/'(_,__/'`\____)
+
+(minikube) $ sudo sysctl fs.inotify.max_user_watches=524288
+(minikube) $ sudo sysctl fs.inotify.max_user_instances=1024
+(minikube) $ sudo mv rktlet rkt stage1-coreos.aci /usr/local/bin/
+(minikube) $ sudo sed -i 's/\(ExecStart.*$\)/\1 --container-runtime=remote --remote-runtime-endpoint=\/var\/run\/rktlet.sock --remote-image-endpoint=\/var\/run\/rktlet.sock/' /etc/systemd/system/localkube.service
+(minikube) $ cat << EOF | sudo tee /etc/systemd/system/rktlet.service
+[Unit]
+Description=rktlet: The rkt implementation of a Kubernetes Container Runtime
+Documentation=https://github.com/kubernetes-incubator/rktlet/tree/master/docs
+
+[Service]
+ExecStart=/usr/local/bin/rktlet
+Restart=always
+StartLimitInterval=0
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
+EOF
+(minikube) $ sudo systemctl daemon-reload
+(minikube) $ sudo systemctl start rktlet
+(minikube) $ sudo systemctl restart localkube
+(minikube) $ exit
+```
+
+You should now have a minikube instance using rktlet!
+
+```shell
+$ kubectl cluster-info dump | grep ContainerRuntime
+                    "ContainerRuntimeVersion": "rkt://0.1.0",
+$ kubectl run --image nginx nginx
+$ kubectl get pods
+NAME                    READY     STATUS    RESTARTS   AGE
+nginx-7c87f569d-ljj8b   1/1       Running   0          30s
+$ kubectl expose deployment nginx --port=80 --type=NodePort
+service "nginx" exposed
+$ kubectl get services
+NAME         TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+kubernetes   ClusterIP   10.0.0.1     <none>        443/TCP        4m
+nginx        NodePort    10.0.0.21    <none>        80:31777/TCP   12s
+$ curl $(minikube ip):31777
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
```

Comment:
The nginx output looks also too verbose. Below this line "Welcome to nginx", could we just skip?